satpy/readers/fci_l1c_nc.py

```diff
@@ -204,22 +220,42 @@ def end_time(self):
         """Get end time."""
         return self.filename_info['end_time']
 
+    def get_channel_measured_group_path(self, channel):
+        """Get the channel's measured group path."""
+        if self.filetype_info['file_type'] == 'fci_l1c_hrfi':
+            channel += '_hr'
+        measured_group_path = 'data/{}/measured'.format(channel)
+
+        return measured_group_path
+
     def get_segment_position_info(self):
-        """Get the vertical position and size information of the chunk (aka segment) for both 1km and 2km grids.
+        """Get information about the size and the position of the chunk inside the final image array.
 
-        This is used in the GEOVariableSegmentYAMLReader to compute optimal chunk sizes for missing chunks.
+        As the final array is composed by stacking chunks (aka segments) vertically, the position of a chunk
+        inside the array is defined by the numbers of the start (lowest) and end (highest) row of the chunk.
+        The row numbering is assumed to start with 1.
+        This info is used in the GEOVariableSegmentYAMLReader to compute optimal chunk sizes for missing chunks.
```

Comment:
Here I'm not anymore sure if we are talking about optimal Dask chunk size or segments :joy: 
```suggestion
        This info is used in the GEOVariableSegmentYAMLReader to compute optimal sizes for missing segments.
```

Reply:
Fixed, see  d73bdde